px-app-nav-group.html

```diff
@@ -0,0 +1,486 @@
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../iron-dropdown/iron-dropdown.html" />
+<link rel="import" href="../neon-animation/neon-animations.html">
+<link rel="import" href="../neon-animation/neon-animation-behavior.html">
+<link rel="import" href="../neon-animation/neon-animation-runner-behavior.html">
+
+<link rel="import" href="px-app-nav-item.html" />
+
+
+<link rel="import" href="css/px-app-nav-group-styles.html">
+
+<dom-module id="px-app-nav-group">
+  <template>
+    <style include="px-app-nav-group-styles"></style>
+    <px-app-nav-item
+        id="itemButton"
+        cancel-select
+        icon="{{icon}}"
+        label="{{label}}"
+        dropdown="{{!hideDropdownIcon}}"
+        selected$="{{_isGroupSelected(selected, noHighlight)}}"
+        overflowed$="{{overflowed}}"
+        empty$="{{empty}}">
+    </px-app-nav-item>
+
+    <div class="app-nav-group__target" id="grouptarget"></div>
+
+    <iron-dropdown
+        id="group"
+        no-overlap
+        dynamic-align
+        opened="{{_opened}}"
+        position-target="[[_getPositionTarget()]]"
+        on-iron-overlay-canceled="_handleGroupCanceled"
+        open-animation-config="{{_openAnimation}}"
+        close-animation-config="{{_closeAnimation}}">
+      <div class$="dropdown-content app-nav-group__dropdown {{_getGroupClasses(superlist)}}" id="groupcontainer">
+        <div class="app-nav-group__dropdown__content" id="groupcontent">
+          <content select="*"></content>
+        </div>
+      </div>
+    </iron-dropdown>
+  </template>
+</dom-module>
+
+<script>
+(function(){
+  Polymer({
+    is: 'px-app-nav-group',
+
+    behaviors: [
+      Polymer.NeonAnimationRunnerBehavior
+    ],
+
+    properties: {
+      /**
+       * If `true`, one of the group's subitems is selected.
+       *
+       * Watch for updates to find out when the group becomes selected through
+       * user interaction or through automatic selection.
+       *
+       * Set the `selected` attribute on one of the group's subitems to select
+       * this group.
+       */
+      selected: {
+        type: Boolean,
+        value: false,
+        notify: true,
+        reflectToAttribute: true,
```

Comment:
This one we do need to reflect (on `path` we don't). We use `selected` to style with CSS, so need to ensure property changes through data binding trigger the attribute to update so the DOM picks up the changes.